tool/src/org/antlr/v4/codegen/target/GoTarget.java

```diff
@@ -65,6 +73,40 @@ protected void addBadWords() {
 	}
 
 	@Override
+	protected void genFile(Grammar g, ST outputFileST, String fileName) {
+		super.genFile(g, outputFileST, fileName);
+		if (DO_GOFMT && !fileName.startsWith(".") /* criterion taken from gofmt*/ && fileName.endsWith(".go")) {
```

Comment:
We want to format just the generated files, not all .go files, right? That might interfere with unrelated .go files.